rfcs/20201102-convert-verbose-into-benchdnn-problem/README.md

```diff
@@ -0,0 +1,50 @@
+# RFC: Convert DNNL_VERBOSE output into benchdnn problems
+
+## Motivation
+
+There are several requests from users and developers to enable DNNL_VERBOSE
+output be "runnable" from benchdnn. Such requests mostly consider the following
+scenario: there is some integration of oneDNN into some framework, and there is
+some topology to run. If validation shows performance or correctness regression,
+it is very hard and/or time consuming to narrow down the specific guilty layer,
+and those users usually send a log file with DNNL_VERBOSE output, which is
+really massive. Manual conversion of these outputs into benchdnn problems takes
+enormous amount of time, since the topology usually consists of 100+ layers. It
+is just inconvenient not to have some tool that allows to perform such
+conversion very fast.
+
+## Tool Requirements
+
+* The tool should accept either a single line or a text file with lines.
+* The output should be fully benchdnn-compatible.
+* The tool should be easy to use.
+
+## Proposal
+
+### Option 1 (recommended) - Python script.
+
+Use the power of Python to create such tool. It will take the input and produce
+benchdnn batch files separated by drivers. Script advantages:
+* Flexibility in terms of features support.
+* Not working script will not affect a release.
```

Comment:
I wouldn't make it a bullet, because we could say something similar about the second option as well. However If we add a script, which will be used by both users and developers, it makes sense to add it into testing and treat it as a feature, which affects release. I agree that priority of an issue caused by not working script can be low.

Reply:
I think the point here is that the issue will not affect release criteria, because if does it cannot be low priority and should be fixed.

Since this serves as an additional tool for debugging convenience, I'm OK with this bullet.

Reply:
I meant what Denis wrote. I've tried to form a sentence in a better way to explain it.